Week1/Homework/js-exercises/Log-the-string.js

```diff
@@ -0,0 +1,8 @@
+'use-strict';
+
+var myString = 'Mohammad Saadaldeen';
```

Comment:
Please use moderen variable declaration `let` or `const`. Let for values that are being reassigned, so for instance in this case `myString` gets reassigned with "Welcome home". If a variable shouldn't be reassigned use a `const` declaration!